.github/workflows/ci.yml

```diff
@@ -64,20 +64,22 @@ jobs:
           python-version: ${{ matrix.python-version }}
           allow-prereleases: true
 
+      - name: Install uv
+        run: python -m pip install --upgrade uv
```

Comment:
This is split out into its own step as "Install old NumPy" doesn't run for some jobs.